src/mavlink_server.cpp

```diff
@@ -23,46 +23,186 @@
 #include "log.h"
 #include "mainloop.h"
 #include "mavlink_server.h"
+#include "util.h"
 
 #define DEFAULT_MAVLINK_PORT 14550
+#define DEFAULT_SYSID 42
```

Comment:
Not really. Just any number different from 1, 255 and any other system ID being used. There is a way to set it in config file doing:

    [mavlink]
    system_id = 123 #change it to 123